src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -303,6 +303,11 @@ def _sanitize_parameters(
                     " https://github.com/huggingface/transformers/pull/20104. Ignore this warning with pipeline(...,"
                     " ignore_warning=True)"
                 )
+            elif self.type == "seq2seq_whisper" and not ignore_warning:
+                logger.warning(
+                    "Using `chunk_length_s` with Whisper models is not recommended and will result in unreliable results, as it uses it's own chunking mechanism "
+                    "(cf. Whisper original paper, section 3.8. Long-form Transcription)."
```

Comment:
As I mentioned offline would be a pity to not use that batch algo in some cases! But up to debate!